pom.xml

```
@@ -69,12 +69,12 @@ limitations under the License.
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>3.55</version>
+        <version>4.76</version>
         <relativePath />
     </parent>
 
     <artifactId>ironmq-notifier</artifactId>
-    <version>1.0.20-SNAPSHOT</version>
+    <version>1.0.19</version>
```

The version value needs to be kept as its previous value.
```suggestion
    <version>1.0.20-SNAPSHOT</version>
```